hail_batch_dev.md

```diff
@@ -192,7 +192,9 @@ Whenever we make a change that isn't purely specific to CPG (like deployment set
 
 ## Deploying changes to production
 
-After a change has been merged to the `main` branch, it can be deployed to the `default` namespace using the `prod_deploy` API endpoint. This will always use the current `HEAD`. Similar to a `dev deploy`, you can specify the steps from `build.yaml` that should be run. Unless there's a good reason to only deploy a particular service, you should use the set listed below. This is a partial list of steps that is specific to the CGP setup, and excludes services we don't use, for example the blog or image fetcher.
+After a change has been merged to the `main` branch, it is automatically deployed to the `default` namespace by calling the `prod deploy` API endpoint from a [GitHub workflow](https://github.com/populationgenomics/hail/blob/main/.github/workflows/prod_deploy.yaml). You should therefore only rarely have to start a `prod deploy` manually.
+
+`prod deploy` will always use the current `HEAD`. Similar to a `dev deploy`, you can specify the steps from `build.yaml` that should be run. Unless there's a good reason to only deploy a particular service, you should use the set listed below. This is a partial list of steps that is specific to the CPG setup, and excludes services we don't use, for example the blog or image fetcher.
 
 It's a good idea to give a quick heads-up in the `#team-software` channel before you're doing this, just in case something breaks.
```

Comment:
```suggestion
```

Might be worth removing this line, because I don't think it's reasonable give a heads-up anytime a hail PR is merged.